src/bin/cmd/wallet.rs

```diff
@@ -44,6 +44,29 @@ pub fn wallet_command(wallet_args: &ArgMatches<'_>, config: GlobalWalletConfig)
 	let wallet_config = config.members.unwrap().wallet;
 
 	let node_client = HTTPNodeClient::new(&wallet_config.check_node_api_http_addr, None);
+
+	// TODO: Very temporary code to obsolete grin wallet for the first hard fork
+	// All tx operations call get_chain_height as a first order of business,
+	// so this is the most non-intrusive place to put this
+	match node_client.clone().chain_height() {
```

Comment:
There's no `node_api_secret` parameter on line 46 when creating a new node_client (`None` there), here the `chain_height()` calling will fail if node API need secret.